packages/crayons-core/package.json

```diff
@@ -60,13 +60,16 @@
     "storybook": "run-p local storybook:start",
     "test:watch": "stencil test --spec --e2e --watchAll --verbose",
     "test": "stencil test --spec --e2e",
-    "test:ci": "stencil test --spec --ci --e2e --max-workers=2"
+    "test:ci": "stencil test --spec --ci --e2e --max-workers=2",
+    "directional-to-logical": "npx postcss ./src/components/**/*.scss ./src/components/**/**/**/*.scss --config ./plugins/postcss-plugins/postcss.config.js --replace --no-map --verbose"
```

Comment:
Can you pls reorganize this npm script alphabetically?

Reply:
Done.